website/docs/best-practices/how-we-mesh/mesh-2-who-is-dbt-mesh-for.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Who is dbt Mesh for?
+description: Understanding if dbt Mesh is the right fit for your team
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+Before embarking on a dbt Mesh implementation, it's important to understand if dbt Mesh is the right fit for your team. Below, we outline three common organizational structures to help teams identify whether dbt Mesh might fit your organization's needs. 
+
+### 1. The Enterprise Data Mesh
+
+Some data teams operation on a global scale -- the team by definition needs to manage, deploy and distribute data products across an incredibly large number of teams. Central IT may own some data products, or simply own the platform upon which data products are built. Often, these organziations have “Architects” who can advise line-of-business teams on their work, while keeping track of what’s happening globally (in terms both of tooling and the substance of work). This is a lot like how software organizations work, beyond a certain scale.
```

Comment:
```suggestion
Some data teams operate on a global scale. By definition, the team needs to manage, deploy, and distribute data products across a large number of teams. Central IT may own some data products or simply own the platform upon which data products are built. Often, these organizations have “architects” who can advise line-of-business teams on their work while keeping track of what’s happening globally (regarding tooling and the substance of work). This is a lot like how software organizations work beyond a certain scale.
```